controllers/utils/validate.ts

```diff
@@ -0,0 +1,37 @@
+import emailValidator from 'email-validator';
+
+export const email = (email: any) => {
+    return emailValidator.validate(email);
+}
+
+export const noneIsBlank = (Obj: any, neededBodyKeys: any) => {
+
+    for(let key of neededBodyKeys) {
+        if (String(Obj[key]).length == 0) {
+            return false;
+        }
+    }
+    return true;
+}
+
+export const countOfKeysMatch = (Obj: any, neededKeys: any) => {
+
+    const countOfKeys = Object.keys(Obj).length;
+    return neededKeys.length <= countOfKeys; 
+
+}
+
+
+
+export const bodyVsKeys = (reqBody: any, neededKeys: any) => {
+
+    // If request body is blank
+    if (Object.keys(reqBody).length === 0 ) {
+        return {reqBody, isValid: false}
+    }
+
+    // Is valid true only if noen of the requeried fields is blank and the count of properties of req.body object is less than equal to the nubmer of needed keys
+    const isValid = noneIsBlank(reqBody,neededKeys) && countOfKeysMatch(reqBody, neededKeys);

```

Comment:
im seeing a lot of misspellings on the comments

Reply:
like the engr manager from paymongo yesterday, he says that let's limit comments and instead use understandable code by making our function and variable names readable and easy to understand